siphon/catalog.py

```diff
@@ -36,6 +36,10 @@ def __getitem__(self, item):
         except TypeError:
             return list(self.values())[item]
 
+    def __repr__(self):
+        """Return useful repr."""
+        return IndexableMapping
```

Comment:
You’re returning the class itself here, which I’m pretty sure is not what you meant.

Reply:
I didn't mean to return a repr here...it was left over from me figuring out how this module worked. I removed it for now.